Production/src/Selectors.cc

```diff
@@ -251,6 +272,7 @@ TauJetSelector::Result TagAndProbe::Select(const edm::Event& event, const std::d
     }
     else { // No mu or ele tag nor high MET 
         tagObject = nullptr;
+        TauJetSelector::selectionType = tau_analysis::selectors::TauJetSelector::Type::GenBased;
```

Comment:
selection type is property of a tau. E.g. in the same event there could be one tau selected by t&p, while few other taus are selected because they are true tau_h